### PR TITLE
1.10.2/sea level

### PR DESCRIPTION
**Pull request by ThelianTech**
This should have all the latest Changes on them.

### CODE REVIEW

File: src/main/java/rtg/api/config/RTGConfig.java [hunk 1]
```diff
@@ -286,12 +286,12 @@ public RTGConfig() {
...
 
-        BASE_TERRAIN_HEIGHT = new ConfigPropertyInt(
+        SEA_LEVEL = new ConfigPropertyInt(
                 ConfigProperty.Type.INTEGER,
                 "Sea Level Base Height",
-                "Sea Level",
+                "Base Terrain Height",
                 "The 'Y' Level that the World Spawns at.",
-                80, 52, 132
+                80, 52, 112
```
Comment (srs-bsns) [inline src/main/java/rtg/api/config/RTGConfig.java]: The default value should be the vanilla default of 63.

File: src/main/java/rtg/api/config/RTGConfig.java [hunk 2]
```diff
@@ -303,7 +303,7 @@ public RTGConfig() {
             "Number of flat bedrock layers",
             "Bedrock",
             "0 = Normal bedrock (rough pattern); 1-5 = Number of flat bedrock layers to generate",
-            0, 0, 5
+            2, 1, 5
```
Comment (srs-bsns) [inline src/main/java/rtg/api/config/RTGConfig.java:306]: Why was this changed?

**Review by srs-bsns — CHANGES_REQUESTED** [review]
`Team-RTG:1.10.2/sea-level` should probably be reset to the state of `Team-RTG:1.10.2-dev`, which should remove the conflicts that are occuring.

File: src/main/java/rtg/api/config/RTGConfig.java [hunk 3]
```diff
@@ -286,6 +283,18 @@
...
         //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        // Base Terrain Height
+        //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+        SEA_LEVEL = new ConfigPropertyInt(
+                ConfigProperty.Type.INTEGER,
+                "Sea Level Base Height",
+                "Base Terrain Height",
+                "The 'Y' Level that the World Spawns at.",
+                80, 52, 112
```
Comment (whichonespink44) [inline src/main/java/rtg/api/config/RTGConfig.java:294]: We can have the default as 80 whilst we're in development, but this will need to be set to 63 when we merge into dev.

File: src/main/java/rtg/api/config/RTGConfig.java (hunk 2 above)
Comment (whichonespink44) [reply src/main/java/rtg/api/config/RTGConfig.java:306]: Yeah, let's keep the default at 0.
